board_rules.md

```diff
@@ -0,0 +1,91 @@
+# Board rules 
```

Comment:
Why some file names are capitalized and some are not?

Reply:
Because as I know, the README.md is usually written with a capital letter. For the names of other files I did not meet restrictions. Shall I renamed other files or it's optional?

Reply:
this is optional in that case, but always be consistent: if you use CAPITALIZED names for md files, you can use such approach for all the files.

Reply:
Thanks, I'll keep it mind for the future.